<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QComboBox" name="comboBox1">
            <item>
             <property name="text">
              <string>Select </string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>cong1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>COM1</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton11">
            <property name="text">
             <string>UHV2</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup1</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton12">
            <property name="text">
             <string>UHV4</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup1</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox1">
            <property name="text">
             <string>On/Off</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>I min</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="MinI1"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>I max</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="MaxI1"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>V min</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="MinV1"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>V max</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="MaxV1"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton1">
            <property name="text">
             <string>Update Range</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QComboBox" name="comboBox2">
            <item>
             <property name="text">
              <string>Select</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton21">
            <property name="text">
             <string>UHV2</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup2</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton22">
            <property name="text">
             <string>UHV4</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup2</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox2">
            <property name="text">
             <string>On/Off</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>I min</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="MinI2"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>I max</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="MaxI2"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>V min</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="MinV2"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>V max</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="MaxV2"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton2">
            <property name="text">
             <string>Update Range</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QComboBox" name="comboBox3">
            <item>
             <property name="text">
              <string>Select</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton31">
            <property name="text">
             <string>UHV2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton32">
            <property name="text">
             <string>UHV4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox3">
            <property name="text">
             <string>On/Off</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>I min</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="MinI3"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>I max</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="MaxI3"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>V min</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="MinV3"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>V max</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="MaxV3"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton3">
            <property name="text">
             <string>Update Range</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="3">
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QComboBox" name="comboBox4">
            <item>
             <property name="text">
              <string>Select</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton41">
            <property name="text">
             <string>UHV2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton42">
            <property name="text">
             <string>UHV4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox4">
            <property name="text">
             <string>On/Off</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>I min</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="MinI4"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_19">
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>I max</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="MaxI4"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_20">
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>V min</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="MinV4"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_21">
            <item>
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>V max</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="MaxV4"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton4">
            <property name="text">
             <string>Update Range</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>418</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup2"/>
  <buttongroup name="buttonGroup1"/>
 </buttongroups>
</ui>
